de.dlr.sc.virsat.project/src/de/dlr/sc/virsat/project/resources/dmf/DmfResource.java

```diff
@@ -98,7 +98,7 @@ protected void saveDMFResource(Map<?, ?> options) throws IOException {
 		// Use NoUndo to suppress notification of this infrastructure-related
 		// operation
 		virSatEd.getVirSatCommandStack().executeNoUndo(dmfResourceSaveCommand);
-		virSatDvlmResource.save(options);
+		virSatEd.saveAll();

```

Comment:
I kind of remember that we changed something like that in the early times of DMF because of ti was messing up the order of sving resources, let me check that it is still working as we expect it.

Reply:
Yes would be great if you could check this. We had two DmfResource Test classes which was creating trouble for whatever reason. Nevertheless saving a dvlm resource outside the editing domain sounds very error prone. If there is an issue with the order of saving we should carefully review what we are doing here.

Reply:
After some intence testing I didn't find any problems.... Let's try it.